### PR TITLE
feat(jira-server): Always show provider row with contact us button

### LINKED ISSUE #6269: Jira Server advertisement for manual setup added to team integrations route
## Issue - Enhancement

When Jira Server v1 ships, we won't have a mechanism for folks to add Jira Server themselves, they'll have to contact us so we can add the provider for them.

We should make this job story as easy as possible:

>  When want to see if Parabol supports Jira Server, I want to see it in the list and have a clear means to add it, so that I can try it out and get going

I suggest we add a ProviderRow to the Integrations page with a CTA that take folks to our contact page with a pre-written subject line to add Jira Server. From there, we can build a relationship with folks and perhaps ask them to sign a trial agreement

### Acceptance Criteria (optional)
Users can:
   - Discover we have Jira Server on the Integrations page
   - Click a button to navigate to a place they can contact us, such as: https://www.parabol.co/contact/?subject=[Jira%20Server]%20Add%20Request

**Estimated effort:** 3 points ([see CONTRIBUTING.md](https://github.com/ParabolInc/parabol/blob/master/CONTRIBUTING.md#points-and-sizes))

### PR DESCRIPTION
**Pull request by igorlesnenko**
Fixes #6269 and adds some markup fixes

image

image

## Testing scenarios

- Create new org and team
- Go to integrations page
- See Jira server row is visible with `contact us` button
- Click `contact us`, see a new tab opened with a contact form with pre-filled subject
- See new event tracked in segment `Clicked Jira Server Request Button`

### CODE REVIEW

**Review by Dschoordsch — APPROVED** [review]
Looks good for me.
@lorena-cure what do you think releasing this row with contact link to the general public?

**Comment by lorena-cure** [comment]
It looks good to me to. I think it's a good way of releasing it despite needing the setup on our end. 
Could I see though what the pre-filled form would look like? Just making sure it's clear for users that they are requesting the integration.

**Comment by igorlesnenko** [comment]
@lorena-cure 
image

**Comment by Dschoordsch** [comment]
@tianrunhe from the analytics point of view, should we pass some utm parameters to the contact page for this?

**Comment by tianrunhe** [comment]
> @tianrunhe from the analytics point of view, should we pass some utm parameters to the contact page for this?

I don't have particular tension on this but I'll let @lorena-cure or @avivapinchas decides.

**Comment by avivapinchas** [comment]
> should we pass some utm parameters to the contact page for this?

Great Q! I believe we should *not*, and here's why: our GA instance runs across both the app and the website, so we can understand how folks are interacting with the totality of our offerings. I believe adding UTM parameters to the link will reset the session, so 2 bad things will happen: 
- We'll double-count that session
- We'll lose the data about where that user originally came from 

To understand whether folks engage with that button, we can do 2 other things: 
- Add a segment event, which we can see in Amplitude 
- In Google Analytics, we can see the previous page that someone viewed before coming to the Contact page 

cc @tghanken to confirm that I actually understand GA though 🙃 

For future ref, UTM parameter standards are [logged in Notion here](https://www.notion.so/parabol/UTM-Parameters-Guidelines-f2e1ada7e80042ff90a304d6511d9e27) 🔒 . 

And one Q while I'm here: is this going to the Contact Us page? I believe it should go to a dedicated Jira Server integration landing page, that has a specific form for this request at the bottom - cc @lorena-cure The reasons for that: we can better gauge interest by looking at traffic to that page, we could customize the form to send some metadata about the request, to make it easier to see that it's a Jira Server request for team members responding to it, and we can gauge interest by looking at form submissions.

File: packages/client/modules/teamDashboard/components/ProviderRow/JiraServerProviderRow.tsx [hunk 1]
```diff
@@ -155,9 +163,32 @@ const JiraServerProviderRow = (props: Props) => {
...
+                {isDesktop ? 'Connect' : <Icon>add</Icon>}
+              </StyledButton>
+            ) : (
+              <Form
+                method='get'
+                target='_blank'
+                action={ExternalLinks.CONTACT}
+                onSubmit={() => {
+                  SendClientSegmentEventMutation(atmosphere, 'Clicked Jira Server Request Button', {
+                    viewerId: id
```
Comment (Dschoordsch) [inline packages/client/modules/teamDashboard/components/ProviderRow/JiraServerProviderRow.tsx:182]: -1 I missed this before, but this is the team member id, not the viewer user id.

**Review by Dschoordsch — CHANGES_REQUESTED** [review]

**Comment by lorena-cure** [comment]
+1 to Aviva's comment. 
I also think it should go to a dedicated Jira Server contact us page, if we have a customized copy and form, it will create less friction. 

I can create this page and form @igorlesnenko

**Comment by igorlesnenko** [comment]
@lorena-cure it would be great, waiting for the new page url, and I'll update my PR

**Comment by tghanken** [comment]
> > should we pass some utm parameters to the contact page for this?
> 
> Great Q! I believe we should _not_, and here's why: our GA instance runs across both the app and the website, so we can understand how folks are interacting with the totality of our offerings. I believe adding UTM parameters to the link will reset the session, so 2 bad things will happen:
> ...

This is correct!  GA picks the latest set of used UTM parameters when calculating attribution, so we should rely on Segment for any additional analytics here.

**Review by Dschoordsch — APPROVED** [review]

**Pull request merged**